Sources/NMeta/Extensions/Request+NMeta.swift

```diff
@@ -13,22 +13,46 @@ public extension Request {
         public let deviceOS: String
         public let device: String
         
+        private enum webEnvironment {
+            static let platform = "web"
+            static let versionString = "0.0.0"
+            static let deviceOS = "N/A"
+            static let device = "N/A"
+        }
+        
         init(request: Request) throws {
             guard let metaString = request.headers.first(name: request.application.nMeta.headerName) else {
                 throw NMetaError.headerMissing
             }
 
             var components = metaString.components(separatedBy: ";")
-
-            guard components.count == 5 else {
+            
+            // As the web environment _does not_ need to have all 5 values we initially checks
+            // that at least two components are present (platform and environment)
+            guard components.count >= 2 else {
                 throw NMetaError.invalidHeaderFormat
             }
-
+            
```

Comment:
Do you have this setting enabled? https://twitter.com/johnsundell/status/952142890020540416

Reply:
I do now! Thanks